PD-1351 / 24.10 / PD-1351 Research and Expand Snapshot and Replication Documentation (#3176)
truenas/documentation: content/SCALETutorials/DataProtection/PeriodicSnapshotTasksSCALE.md

Each has an hourly snapshot taken.
After a week, snapshots created at *01.00* through *23.00* get deleted, but you keep snapshots timed at *00.00* because they are necessary for the second periodic task. 
These snapshots get destroyed at the end of 3 years.

> ### Configuring Snapshot Granularity

Reviewer: Why would a user want to configure this? We should say why.
Second, you don't actually tell them how to configure it, just provide information about it.
If you want to change this to and "about snapshot granularity" header that fits the information contained in this section.
Recommend either changing the header, or adding instructions on how to configure it.

> Heightened snapshot granularity is directly linked to an increase in the rate of snapshots taken. Raising the overall snapshot granularity often leads to the need for lower retention times, as this configuration routinely takes up more space. Similarly, a high level of snapshot granularity tends to decrease system performance overtime due to a raise in processing and indexing needs. Provided this, configuring your system with high snapshot granularity is typically only recommended to users who need detailed recovery options for vital system data.

Reviewer: I don't think the users is going to understand what granularity means in this paragraph. Does it mean you have more snapshots and/or more recursive snapshots taken and stored? I think you can explain what you mean in simple terms so you don't use the reader because they aren't sure what you mean.

> Comparatively, lowering snapshot granularity is done by decreasing the rate that your system takes snapshots. By employing this configuration, less system storage is utilized by the snapshot tasks and overall snapshot retention increases; however, the amount of reliable recovery options becomes limited.

Reviewer: don't use "utilize" kept it simple. maybe this instead
```suggestion
To lower snapshot granularity decrease the rate the system takes snapshots. 
With this configuration the snapshots tasks use less system storage and overall snapshot retention increases; however, this does limit the amount of reliable recovery options.
```

> ### Using Replication Tasks for Cloud Targets

Reviewer: This section does not address how to use replication, or how to set up the cloud targets, so this header is misleading.

> If users intend to utilize a cloud sync task to upload snapshots to a cloud storage provider, decreasing snapshot granularity can provide a method to reduce costs associated with data transfer and cloud storage. 

Reviewer: explain that granularity again. the user might not know what you are talking about. That said, suggest the following simplified wording for what you are saying.
```suggestion
Using a cloud sync task to upload snapshots to a cloud storage provider decreases snapshot granularity and can provide a way to reduce costs associated with data transfer and cloud storage. 
```

> By opting for local snapshot storage, users have a more optimum environment for periodic snapshot tasks with high levels of granularity, as local storage tends to have lower latency and faster access speeds. It is generally recommended that users configure snapshot replication tasks with lower RTO requirements for rapid recovery when implementing local storage.

Reviewer: What is RTO? if you use an acronym, spell it out then use (RTO) behind the spelled out acronym.
Again, using granularity is not really explained in this new content. It is better to just plainly say what it is.
```suggestion
Using local snapshot storage provides a more optimum environment for periodic snapshot tasks with high levels of granularity, as local storage tends to have lower latency and faster access speeds.
We generally recommend configuring snapshot replication tasks with lower RTO requirements for rapid recovery when implementing local storage.
```
